### ISSUE TITLE
Apache mirror no longer exists

### ISSUE THREAD

**Issue by sethfduke** [steveny2k/docker-predictionio#14]
http://mirror.nexcess.net/apache/incubator/predictionio/ cannot be found, nor does Nexcess list predictionio as a mirror it currently maintains on its servers.

**Comment by MaazMohamedy** [comment]
Also to let you know I pulled your repo, re-built your image (had to modify one or two lines as they were out of date) but when I ran `pio status`  I relieved this error:

`Connection refused (java.net.ConnectException)`

Can you please fix this?

@steveny2k

**Comment by steveny2k** [comment]
@MaazMohamedy Thanks for info. May I know if you directly pull the docker image, does it work fine? Thanks

**Comment by MaazMohamedy** [comment]
@steveny2k Yes, it works fine if you pull the docker image. But I need to edit the docker image so I can't just use the one that I pull

**Comment by steveny2k** [comment]
@MaazMohamedy Thanks for your info.
Maybe there are URL changes etc which need to be updated.
Will try to find time to debug and update.
